src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,87 @@
+             Then it should respond with status 200
+              And response body is
+                  """
+                  []
+                  """
+
+        @ignore
+        Scenario: Ensure clients can't GET information when filtering by limit and offset using invalid offset
+             When sending a GET request to "/ed-fi/schools?limit=6&offset=<Value>"
+             Then it should respond with 400
+              And response body is
+                  """
+                  {"detail": "The offset parameter was incorrect.","type": "urn:ed-fi:api:bad-request:parameter","title": "Parameter Validation Failed","status": 400,"correlationId": null,"errors": ["Offset must be omitted or set to a value between 0 and 500."]}
+                  """
+        Examples:
+                  | Value                    |
+                  | -1                       |
+                  | 'zero'                   |
+                  | '5; select * from users' |
+                  | '0)'                     |
+                  | '1%27'                   |
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying using an offset without providing any limit in the query string
+            Given these schools exist:
+                  | schoolId | nameOfInstitution                             | gradeLevels         | educationOrganizationCategories     |
+                  | 5        | School with max edorgId value                 | [ "Postsecondary" ] | [ "School" ]                        |
+                  | 6        | UT Austin College of Education Under Graduate | [ "Postsecondary" ] | [ "Educator Preparation Provider" ] |
+             When sending a GET request to "/ed-fi/schools?offset=1"
+             Then it should respond with 200
+              And response body is
+                  """
+                  {"id":"f24dc237b73741b4a778ce403906d8f9","schoolId":6,"nameOfInstitution":"UT Austin College of Education Under Graduate","addresses":[{"addressTypeDescriptor":"uri://ed-fi.org/AddressTypeDescriptor#Physical","city":"Austin","postalCode":"78712","stateAbbreviationDescriptor":"uri://ed-fi.org/StateAbbreviationDescriptor#TX","streetNumberName":"1912 Speedway Stop D5000","nameOfCounty":"Travis","periods":[]}],"educationOrganizationCategories":[{"educationOrganizationCategoryDescriptor":"uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"}],"identificationCodes":[],"indicators":[],"institutionTelephones":[],"internationalAddresses":[],"_ext":{"tpdm":{"postSecondaryInstitutionReference":{"postSecondaryInstitutionId":6000203,"link":{"rel":"PostSecondaryInstitution","href":"/ed-fi/postSecondaryInstitutions/02f5fec829834b7691db381ff5f61ff1"}}}},"schoolCategories":[],"gradeLevels":[{"gradeLevelDescriptor":"uri://ed-fi.org/GradeLevelDescriptor#Postsecondary"}],"_etag":"5250169253470311260","_lastModifiedDate":"2024-04-10T05:25:04.2923356Z"}
+                  """
```

Comment:
I am going to update the response above:

1. Reformat for easier reading.
2. The "Given" sections don't setup any addresses, therefore the response shouldn't have any addresses. I will remove those and anything else that is not in the given/background.
3. Removing `link`, `_tag`, `_lastModifiedDate`, and `_ext`. These are features that we won't support in the Data Management Service right away (in the case of `link`, we'll never support it).

The last detail in particular I wouldn't have expected either of you to know, because this is the first time it has come up.

On the first point, I see that I have setup a tricky situation: I suggested tables instead of JSON for the Givens because it makes the data entry more compact. The table contains only the required fields. But, here's a problem: you can't just paste that exact body into the ODS/API and see what comes back. That is unfortunate, and we should think about whether we want to change that long-term. But not right now - I just mean thinking about it for a future ticket. Perhaps we can discuss in retrospective.